src/SFA.DAS.PSRService.IntegrationTests/ReportSubmission/Given_I_Have_Created_A_Report/And_I_Have_Reported_All_Mandatory_Factors/And_All_Mandatory_Numbers/When_I_Sumit_The_Report.cs

```diff
@@ -0,0 +1,42 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using NUnit.Framework;
+
+namespace SFA.DAS.PSRService.IntegrationTests.ReportSubmission.Given_I_Have_Created_A_Report.
+    And_I_Have_Reported_All_Mandatory_Factors.And_All_Mandatory_Numbers
+{
+    public sealed class When_I_Sumit_The_Report
+        : And_All_Mandatory_Numbers
+    {
+        private IActionResult submitResponse;
+
+        protected override void When()
+        {
+            submitResponse
+                =
```

Comment:
can we tidy this up as its hard to read at the moment spread over multiple lines